src/en.json

```
@@ -2477,6 +2477,10 @@
         "id": "horseshit",
         "match": "horseshit|horse-shit|horse shit"
       },
+      {
+        "id": "meecrob",
+        "match": "pile of meecrob"
```

This JSON takes pipe-separated values for the `match` attribute and will turn those into individual items in the `en.txt` list; the `id` attribute value isn't added to the `.en.txt` list from this though.

Based on your original change to `en.txt`, this would instead be:
```suggestion
        "match": "meecrob"
```

Otherwise, if you wanted to see the full match for either/or you could use:
```suggestion
        "match": "meecrob|pile of meecrob"
```

Though just defining `meecrob` for the match would probably result in the desired behaviour for most implementations using this list, e.g. `piece of meecrob` -> `piece of *******`